front/src/styles/scss/applications/operationalStudies/_scenario.scss

```diff
@@ -352,27 +352,29 @@
               align-items: center;
               justify-content: right;
 
-              .after-midnight {
-                max-width: 1rem;
-                margin: 0 0.5rem;
-              }
-
-              .scenario-timetable-train-departure {
-                font-weight: 400;
-                font-size: 1rem;
-              }
-
-              .not-honored-or-too-fast {
+              .logo {
                 width: 1.5rem;
                 height: 1.5rem;
-                color: var(--warning60);
+                color: var(--info60);
```

Comment:
Maybe we can just drop this line since it's overridden in both cases?

Reply:
Done